packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,40 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers: []
+icon: ""
```

Comment:
@jiaqiluo or @aiyengar2 do we have a logo for this ? We'll need to add one too: https://github.com/rancher/charts/tree/release-v2.5/assets/logos

Reply:
I will update the icon url once it is available.